blockchain/blockchain.go

```diff
@@ -506,9 +506,12 @@ func (b *BlockChain) CreateCRAssetsRectifyTransaction() (*Transaction, error) {
 	if err != nil {
 		return nil, err
 	}
-	if len(utxos) < int(b.chainParams.MaxCRAssetsAddressUTXOCount) {
+	if len(utxos) < int(b.chainParams.MinCRAssetsAddressUTXOCount) {
```

Comment:
Here should be less than MinCRAssetsAddressUTXOCount.

Reply:
fixed